server/sdk/lib/Events.ts

```diff
@@ -1,28 +1,22 @@
-import { IUIKitInteraction } from '@rocket.chat/apps-engine/definition/uikit';
```

Comment:
Same here, it seems this file has unnecessary removals, maybe you were working on an outdated branch?

Reply:
It was overwritten by mistake. I have fixed the issue in the latest commit.